content/en/blog/_posts/2022-09-14-pod-has-network-condition.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: blog
+title: 'Kubernetes 1.25: PodHasNetwork condition for pods'
+date: 2022-09-14
+slug: pod-has-network-condition
+---
+
+**Author:**
+Deep Debroy (Apple)
+
+Kubernetes 1.25 introduces Alpha support for a new kubelet-managed pod condition
+in the status field of a pod: `PodHasNetwork`. The kubelet will use the
+`PodHasNetwork` condition to accurately surface the initialization state of a
+pod from the perspective of pod sandbox creation and network configuration by a
+CRI plugin in coordination with CNI plugins. The kubelet starts to pull
```

Comment:
Hmmm... that suggestion seems to be a bit of spagetti. CRI means "container runtime interface", so "container runtime" after that acronym seems odd.
Maybe better would be to use `implementation` in place of suggested `container runtime`?

Reply:
Will simplify to `container runtime` instead of `CRI container runtime` for now. After DockerShim deprecation, container runtime in the context of Kubernetes/Kubelet should imply that the container runtime implements CRI.

Reply:
Done